README.rst

```diff
@@ -0,0 +1,315 @@
+========================================================
+REANA example - LHCb Rare Charm Decay Search
+========================================================
+
+.. image:: https://img.shields.io/travis/reanahub/reana-demo-lhcb-d2pimumu.svg
+   :target: https://travis-ci.org/reanahub/reana-demo-lhcb-d2pimumu
+
+.. image:: https://badges.gitter.im/Join%20Chat.svg
+   :target: https://gitter.im/reanahub/reana?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
+
+.. image:: https://img.shields.io/github/license/reanahub/reana-demo-lhcb-d2pimumu.svg
+   :target: https://raw.githubusercontent.com/reanahub/reana-demo-lhcb-d2pimumu/master/LICENSE
+
+
+
+About
+======
+
+This example is based on the
+`analysis-case-study <https://github.com/atrisovic/analysis-case-study>`_.
+
+Analysis structure
+===================
+
+Making a research data analysis reproducible basically means to provide
+"runnable recipes" addressing (1) where is the input data, (2) what software was
+used to analyse the data, (3) which computing environments were used to run the
+software and (4) which computational workflow steps were taken to run the
+analysis. This will permit to instantiate the analysis on the computational
+cloud and run the analysis to obtain (5) output results.
+
+
+1. Input data
+-------------
+The data being used is currently only available on request. It consists of a 10
+GB ROOT file.
+
+2. Analysis code
+----------------
+
+You can either run the analysis step by step (explained below), or use the bash
+script :code:`run.sh` via:
+
+.. code-block:: console
+
+   $ bash run.sh <data_file> <tmp_phi_models> <result_plot>
```

Comment:
`run.sh` is used only once, and it contains only three steps, so it should not be really necessary to make such an abstraction.  It is perhaps better to list the steps directly here, without creating run.sh at all, so that readers can see the steps directly in the README without having to open another tiny script.